### PR TITLE
Add blog.html

### PR DESCRIPTION
**Pull request by honey7100**
I added the html file for the "Blog" section of my website. I created a post and will work on the functionality of the comment section separately. The current css in the file will be modified when I import bootstrap.

### CODE REVIEW

File: portfolio/src/main/webapp/blog.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <title>Oyin's Blog</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand&display=swap" rel="stylesheet">
+</head>
+
+<style>
+    body{
+        font: 22px 'Quicksand', sans-serif;
+        background-color: #a8d0e6;
+        }
```
Comment (codewithjess) [inline portfolio/src/main/webapp/blog.html]: The spacing is a little off here

```suggestion
    body {
        font: 22px 'Quicksand', sans-serif;
        background-color: #a8d0e6;
    }
```

File: portfolio/src/main/webapp/blog.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+</style>
+
+
+<body>
+    <h1>"How Grace Hopper Impacted Me"</h1>
+    <h2>By: Oyin Sholeye</h2>
+    <h3>Posted on May 28, 2020</h3>
+    <img src="/images/IMG-3018.JPG" alt="GHC" style="width: 250px;height:300px" >
```
Comment (codewithjess) [inline portfolio/src/main/webapp/blog.html:19]: You can upload the image file in this pull request too so its not missing!

File: portfolio/src/main/webapp/blog.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand&display=swap" rel="stylesheet">
+</head>
+
+<style>
+    body{
+        font: 22px 'Quicksand', sans-serif;
+        background-color: #a8d0e6;
+        }
+</style>
+
```
Comment (codewithjess) [inline portfolio/src/main/webapp/blog.html]: nit: extra line not needed

File: portfolio/src/main/webapp/blog.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        <p> The main reasons why Grace Hopper was so influencial:</p>
+        <p> 1. I was able to interview with Google at the conference which led to me getting and accepting 
+            and internship offer for STEP.</p>
+        <p>2. I was able to build new friendships with some wonderful girls.</p>
+
+        <form>
+            <label for="comment"> Comments or Questions:</label><br>
+            <input type="text" id="comment" name="comment" >
+            <input type="submit" value="Post">
+        </form>
```
Comment (codewithjess) [inline portfolio/src/main/webapp/blog.html]: Looks like this part got indented an extra tab. Lets keep it at the same indentation as the <p> on line 23.

File: portfolio/src/main/webapp/blog.html [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+</body>
+
+</html>
```
Comment (codewithjess) [inline portfolio/src/main/webapp/blog.html]: File is missing a newline at the end

**Review by codewithjess — COMMENTED** [review]
Nice job overall! Just left some minor comments.

Also, looks like Google's style guide recommends to indent by 2 spaces (instead of 4 spaces). [go/htmlcssstyle#Indentation](go/htmlcssstyle#Indentation)

File: portfolio/src/main/webapp/blog.html (hunk 1 above)
Comment (honey7100) [reply portfolio/src/main/webapp/blog.html]: I will make sure to keep it at 2 indents and fix the whole thing, thank you.

File: portfolio/src/main/webapp/blog.html (hunk 2 above)
Comment (honey7100) [reply portfolio/src/main/webapp/blog.html:19]: IMG-3018

I am hoping this shows up.

File: portfolio/src/main/webapp/blog.html (hunk 3 above)
Comment (honey7100) [reply portfolio/src/main/webapp/blog.html]: Will remove the line

File: portfolio/src/main/webapp/blog.html (hunk 4 above)
Comment (honey7100) [reply portfolio/src/main/webapp/blog.html]: I will move the lines over to align with line 23 .

File: portfolio/src/main/webapp/blog.html (hunk 5 above)
Comment (honey7100) [reply portfolio/src/main/webapp/blog.html]: Does that mean create a blank line on 54?

File: portfolio/src/main/webapp/blog.html (hunk 2 above)
Comment (rharagutchi) [reply portfolio/src/main/webapp/blog.html:19]: I think Jessica meant adding the image as a file in your branch :) 

This pull request currently only has your HTML file in it, you'll also want your image file to be a file in the request

**Review by rharagutchi — COMMENTED** [review]
Is the plan to merge both pull requests or was that other one just for testing?

File: portfolio/src/main/webapp/blog.html (hunk 1 above)
Comment (codewithjess) [reply portfolio/src/main/webapp/blog.html]: Thanks :) one tiny nit - there should be a space after body: `body {`

File: portfolio/src/main/webapp/blog.html (hunk 2 above)
Comment (codewithjess) [reply portfolio/src/main/webapp/blog.html:19]: Yeah what Rodrigo said :)

File: portfolio/src/main/webapp/blog.html (hunk 5 above)
Comment (codewithjess) [reply portfolio/src/main/webapp/blog.html]: Correct, https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline for more context

**Review by rharagutchi — APPROVED** [review]

**Review by codewithjess — APPROVED** [review]

**Pull request merged**